src/main.rs

```diff
@@ -192,29 +210,56 @@ fn print_usage(program: &str, opts: Options) {
     print!("{}", opts.usage(&brief));
 }
 
+fn load_config(config_file: Option<String>) -> Config {
+    let f = match config_file {
+        Some(f) => f,
+        None => {
+            let default = "config.toml";
+            if !Path::new(default).exists() {
+                return Config::default();
+            }
+            default.to_string()
```

Comment:
I'm not sure how I feel about implicitly using a `config.toml`, but if we do then we should probably at least print that we are.